### PR TITLE
Adding PATCH support

### PR DESCRIPTION
**Pull request by EricAnderson1000**
https://github.com/mmazi/rescu/issues/95

@mmazi Reviewer

### CODE REVIEW

**Comment by EricAnderson1000** [comment]
>Unsupported major.minor version 52.0

Is support of JDK 7 required?

This upgrade might be difficult
```
 <groupId>javax.ws.rs</groupId>
 -            <artifactId>jsr311-api</artifactId>
 -            <version>1.1.1</version>
 +            <artifactId>javax.ws.rs-api</artifactId>
 +            <version>2.1</version>
```

**Comment by mmazi** [comment]
Thanks for the PR!

Yes, Java 7 support is a requirement. Can this be done with Java 7?

File: pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
     <groupId>com.github.mmazi</groupId>
     <artifactId>rescu</artifactId>
-    <version>1.9.1-SNAPSHOT</version>
+    <version>1.9.1-PATCH-SNAPSHOT</version>
```
Comment (mmazi) [inline pom.xml:30]: This change should be removed for PR to be merged.

**Comment by mmazi** [comment]
Re-opened as #97

**Pull request closed without merging**